services/QuillLMS/app/controllers/api/v1/activity_sessions_controller.rb

```diff
@@ -94,7 +95,9 @@ def destroy
                   :activity_uid,
                   :activity_id,
                   :anonymous,
-                  :temporary)
+                  :temporary,
+                  :data
```

Comment:
I' actually not sure how this would work given line 87 above.  I presume that it does since you've done testing, but maybe if we're going to access the value from `params` we should remove the `@data = params.delete(:data)` line since it provides a second path to access the data?

Reply:
You're right, I think this addition actually isn't doing anything - data comes back in the `.merge` below but is never present at this point. I'll remove it.